sopel/coretasks.py

```diff
@@ -1045,6 +1045,11 @@ def auth_proceed(bot, trigger):
     """
     if trigger.args[0] != '+':
         # How did we get here? I am not good with computer.
+        LOGGER.info(
+            "Aborting SASL: unexpected server reply '%s'" % trigger)
```

Comment:
My local branch has changed this to a warning per #2191, but I'm not going to waste more Travis time on this PR until I can look into that `SSLError`. Once that's fixed, I'll force-push.